public_html/logs/index.php

```diff
@@ -21,17 +21,15 @@
      $selected_channel = $_GET['display'];
 
 $html = new HtmlPage("Wikimedia IRC logs browser");
-googlefonts_init($html);
 $html->Style->items['*']['font-family'] = 'Open Sans';
 
 // Github ribbon
-$github_ribbon = new GitHub_Ribbon();
-$github_ribbon->Repository = "benapetr/wikimedia-bot/tree/master/public_html/logs";
-$html->AppendHtml($github_ribbon->ToHtml());
+$html->AppendHtml('<a class="github-fork-ribbon right-top" href="https://github.com/benapetr/wikimedia-bot/tree/master/public_html/logs" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>');
```

Comment:
Isn't github.com an external site?

Reply:
It's in a href, not script or link or img that are "assets" and will not automatically loaded.